src/customTypes.d.ts

```diff
@@ -68,3 +68,22 @@ declare module 'mozlog' {
 interface Window {
   gtag: any
 }
+
+declare namespace breach {
```

Comment:
If the types agree with the schema that sounds good, but maybe change it to be an actual type, instead of a namespace:

```ts
type Breach = {
  Name: string;
  Title: string;
  // ...
}
```

Other than that this looks good to go!